frontends/verific/verific.cc

```diff
@@ -450,8 +450,30 @@ void VerificImporter::import_attributes(dict<RTLIL::IdString, RTLIL::Const> &att
 		auto type_range = nl->GetTypeRange(obj->Name());
 		if (!type_range)
 			return;
+		if (type_range->IsTypeScalar()) {
+		  const long long left_bound = type_range->GetScalarRangeLeftBound();
```

Comment:
I suggest bottom and top bound, or some other naming. I assume it's always the case `left_bound <= right_bound`

Reply:
Actually, if we put a range such as:
        `a : in INTEGER range 15 to -15;`
Verific gives the following warning:

> VERIFIC-WARNING [VHDL-1200] negative_integer_range.vhdl:7: range is empty (null range)

But continues.
This results in a completely incoherent RTLIL:

  ```
attribute \bottom_bound 2'11
  attribute \is_bound_signed 1'1
  attribute \src "negative_integer_range.vhdl:7.9-7.10"
  attribute \top_bound 2'01
```

Which corresponds with a [-1, 1] decimal range.
This range has nothing to do with the original (wrong) range nor with the Verific warning.

Anyway, I changed to `bottom_bound` and `top_bound` in last commit. Thanks!